csharp/ql/lib/change-notes/2023-01-03-genericattributes.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added library support for generic attributes.
```

Comment:
I would also add that `cil.ConstructedType::getName` was changed. If anyone relied on that it's a breaking change, isn't it?

Reply:
That is a good point! According to the [change note description](https://github.com/github/codeql/blob/main/docs/change-notes.md) for libraries, the category is probably `majorAnalysis`.